src/router/storeSync/storeSync.config.js

```diff
@@ -42,6 +42,7 @@ const storeSyncConfig = [
     // to the default projection EPSG number
     new PositionParamConfig(),
     new CameraParamConfig(),
+    new LayerParamConfig(),
```

Comment:
Here we should add a comment why the layer param needs to be before z param, a comment related to the `zoomToExtent` in LayerParamConfig. But consider also trying first using `from` instead of `to`